### PR TITLE
Galactic to main

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,24 +104,47 @@ We should include:
...
+	<child link="finger_left"/>
+	<limit effort="1000.0" lower="0" upper="0.38" velocity="10"/>
+</joint>
+
+<joint name="right_finger_joint" type="prismatic">
+	<axis xyz="0 1 0"/>
+	<origin xyz="0.0 -0.48 1" rpy="0.0 0.0 0.0"/>
+	<parent link="base"/>
+	<child link="finger_right"/>
+	<limit effort="1000.0" lower="0" upper="0.38" velocity="10"/>
```
Comment (destogl) [inline README.md:120]:
```suggestion
  <mimic joint="right_finger_joint"/>
  <axis xyz="0 1 0"/>
  <origin xyz="0.0 0.48 1" rpy="0.0 0.0 3.1415926535"/>
  <parent link="base"/>
  <child link="finger_left"/>
  <limit effort="1000.0" lower="0" upper="0.38" velocity="10"/>
</joint>

<joint name="right_finger_joint" type="prismatic">
  <axis xyz="0 1 0"/>
  <origin xyz="0.0 -0.48 1" rpy="0.0 0.0 0.0"/>
  <parent link="base"/>
  <child link="finger_right"/>
  <limit effort="1000.0" lower="0" upper="0.38" velocity="10"/>
```

File: README.md [hunk 2]
```diff
@@ -100,24 +104,47 @@ We should include:
...
+	<joint name="left_finger_joint">
+		<param name="mimic">right_finger_joint</param>
+		<param name="multiplier">1</param>
+		<command_interface name="position"/>
+		<state_interface name="position"/>
+		<state_interface name="velocity"/>
+		<state_interface name="effort"/>
+	</joint>
+
+</ros2_control>
```
Comment (destogl) [inline README.md:147]:
```suggestion
<ros2_control name="GazeboSystem" type="system">

  <hardware>
    <plugin>gazebo_ros2_control/GazeboSystem</plugin>
  </hardware>

  <joint name="right_finger_joint">
    <command_interface name="position"/>
    <state_interface name="position"/>
    <state_interface name="velocity"/>
    <state_interface name="effort"/>
  </joint>

  <joint name="left_finger_joint">
    <param name="mimic">right_finger_joint</param>
    <param name="multiplier">1</param>
    <command_interface name="position"/>
    <state_interface name="position"/>
    <state_interface name="velocity"/>
    <state_interface name="effort"/>
  </joint>

</ros2_control>
```

File: gazebo_ros2_control/CHANGELOG.rst [hunk 3]
```diff
@@ -32,10 +32,32 @@ Changelog for package gazebo_ros2_control
...
+  Co-authored-by: Keegan Sotebeer <ksotebeer95@gmail.com>
+  Co-authored-by: Maciej Bednarczyk <83034299+mcbed@users.noreply.github.com>
+  Co-authored-by: Bence Magyar <bence.magyar.robotics@gmail.com>
+* Contributors: Alejandro Hernández Cordero, Leander Stephen D'Souza
+
+0.1.0 (2022-05-27)
+------------------
+* Declare dependency of gazebo_hardware_plugins to urdf in CMakeLists.txt (`#117 <https://github.com/ros-simulation/gazebo_ros2_control/issues/117>`_) (`#119 <https://github.com/ros-simulation/gazebo_ros2_control/issues/119>`_)
+  Co-authored-by: Martin Wudenka <Martin.Wudenka@gmx.de>
+
```
Comment (destogl) [inline gazebo_ros2_control/CHANGELOG.rst:55]: Not sure if it is good idea to make changes to CHANGELOG file...

```suggestion
```

File: gazebo_ros2_control/CHANGELOG.rst [hunk 4]
```diff
@@ -32,10 +32,32 @@ Changelog for package gazebo_ros2_control
...
 0.0.8 (2022-01-28)
 ------------------
 * Enable setting default position of the simulated robot using ros2_control URDF tag. (`#100 <https://github.com/ros-simulation/gazebo_ros2_control//issues/100>`_)
 * Contributors: Denis Štogl
+=======
```
Comment (destogl) [inline gazebo_ros2_control/CHANGELOG.rst:60]:
```suggestion
```

File: gazebo_ros2_control_demos/CHANGELOG.rst [hunk 5]
```diff
@@ -27,6 +27,25 @@ Changelog for package gazebo_ros2_control_demos
...
+* [backport Foxy] `#136 <https://github.com/ros-controls/gazebo_ros2_control/issues/136>`_ `#147 <https://github.com/ros-controls/gazebo_ros2_control/issues/147>`_ `#139 <https://github.com/ros-controls/gazebo_ros2_control/issues/139>`_ `#134 <https://github.com/ros-controls/gazebo_ros2_control/issues/134>`_ (`#150 <https://github.com/ros-controls/gazebo_ros2_control/issues/150>`_)
+  Co-authored-by: Keegan Sotebeer <ksotebeer95@gmail.com>
+  Co-authored-by: Maciej Bednarczyk <83034299+mcbed@users.noreply.github.com>
+  Co-authored-by: Bence Magyar <bence.magyar.robotics@gmail.com>
+* Contributors: Alejandro Hernández Cordero, Bence Magyar, Leander Stephen D'Souza
+
+0.1.0 (2022-05-27)
+------------------
+* Added diff drive example (`#113 <https://github.com/ros-simulation/gazebo_ros2_control/issues/113>`_)
+
```
Comment (destogl) [inline gazebo_ros2_control_demos/CHANGELOG.rst:48]:
```suggestion
```

**Review by destogl — CHANGES_REQUESTED** [review]
A few minor comments.

**Comment by ahcorde** [comment]
Good question @bmagyar, I will clean the conflicts or close it if this is not relevant

**Pull request closed without merging**